.changeset/rich-rules-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Added a new `small` size variant to `Label`
```

Comment:
Nit to avoid using 'variant'. A link to the example might be nice here too.

```suggestion
Added a new `small` size option for `Label`

🔗 [See example](https://primer.style/brand/components/Label/react#sizes)
```